tests/integration/goldens/asset/samples/generated_samples/snippet_metadata_v1_asset_v1.json

```diff
@@ -0,0 +1,1048 @@
+{
```

Comment:
For the time beeing, we dropped versioning of the metadata scheam so you can drop the first v1 here.
Also, we are doing, where possible, snippet_metadata_{protopackage}.json, although the real important part is snippet_metadata*.json as the rest of the name is not going to be used to identify anything automatically.